solutions/arrays/alternate-sort.js

```diff
@@ -0,0 +1,61 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let testCases = parseInt(readLine());
+while (testCases > 0) {
+    let arr = readLine().split(" ").map(Number);
+    let evenPositionArray = [];
+    let oddPositionArray = [];
+    // storing values in even and odd indexing based arrays
+    for (let index = 0; index < arr.length; index++) {
+        if (index % 2 === 0) {
+            evenPositionArray.push(arr[index]);
+        }
+        else {
+            oddPositionArray.push(arr[index]);
+        }
+    }
+    // sorting odd index based array in ascending order using bubble sort

```

Comment:
@DevratAgarwal We don't teach `bubble sort` in the bootcamp. So, please use `insertion sort`.

Reply:
@sailesh-ramayanam done with insertion sort.